index.src.html

```diff
@@ -1168,10 +1207,11 @@ <h4 algorithm id="collectfromcredentialstore-passwordcredential">
   </ol>
 
   <h4 algorithm id="create-passwordcredential">
-    `PasswordCredential`'s `[[Create]](options)`
+    `PasswordCredential`'s `[[Create]](origin, options)`
   </h4>
 
-  <dfn for="PasswordCredential" method>\[[Create]](options)</dfn> is called with a
+  <dfn for="PasswordCredential" method>\[[Create]](origin, options)</dfn> is called with
+  an [=environment settings object/origin=] (<var ignore>origin</var>) and a
```

Comment:
I think you need to thread this through to https://w3c.github.io/webappsec-credential-management/#construct-passwordcredential-data and https://w3c.github.io/webappsec-credential-management/#construct-passwordcredential-form.

Reply:
ok, will look into it.

Reply:
This still needs looking-into, right?

Reply:
yep

Reply:
an attempt at doing such is here: 6bdadaa